typescript/infra/config/environments/testnet3/funding.ts

```diff
@@ -0,0 +1,25 @@
+import { KEY_ROLE_ENUM } from '../../../src/agents/roles';
+import { ConnectionType } from '../../../src/config/agent';
+import { KeyFunderConfig } from '../../../src/config/funding';
+import { Contexts } from '../../contexts';
+
+import { environment } from './chains';
+
+export const keyFunderConfig: KeyFunderConfig = {
+  docker: {
+    repo: 'gcr.io/abacus-labs-dev/hyperlane-monorepo',
+    tag: 'sha-76c3f13',
+  },
+  // We're currently using the same deployer key as testnet2.
+  // To minimize nonce clobbering we offset the key funder cron
+  // schedule by 30 minutes.
```

Comment:
😬

Reply:
Don't judge me

Reply:
But yeah I think this is pretty easy to adjust later if we want to